.github/workflows/build.yml

```diff
@@ -795,7 +795,7 @@ jobs:
     needs: [job_get_metadata, job_build]
     runs-on: ubuntu-20.04
     timeout-minutes: 30
-    if: contains(github.event.pull_request.labels.*.name, 'ci-overhead-measurements')
+    if: contains(github.event.pull_request.labels.*.name, 'ci-overhead-measurements') || github.ref == 'refs/heads/master'
```

Comment:
You can use `needs.job_get_metadata.outputs.is_master == 'true'` here instead!

Reply:
ok, changed